project/build.sbt

```diff
@@ -17,7 +17,8 @@ libraryDependencies ++= Seq(
   "org.eclipse.jgit" % "org.eclipse.jgit.pgm" % "3.2.0.201312181205-r"
 )
 
-addSbtPlugin("org.portable-scala" % "sbt-platform-deps" % "1.0.0-M2")
+addSbtPlugin("org.portable-scala" % "sbt-platform-deps" % "1.0.0")
+addSbtPlugin("org.portable-scala" % "sbt-crossproject"  % "0.6.0")
```

Comment:
It's a bit unfortunate that we have to depend on  cross project here, but I don't have an easy fix for it.